src/utils/filename-utils.ts

```diff
@@ -17,10 +17,14 @@ export const normalizeTitle = (title: string) => {
 };
 
 export const getFileIndex = (dstPath: string, fileNamePrefix: string): number | string => {
-
   const index = fs
     .readdirSync(dstPath)
-    .filter(file => file.indexOf(fileNamePrefix) > -1)
+    .filter(file => {
+      // make sure we get the first copy with no count suffix or the copies whose filename changed
+      const filePrefix = file.split('.')[0];
```

Comment:
hmm.. I think it won't work, if the fileNamePrefix contains a '.' character, because in this case the file.split returns with the first part of the the current file, while fileNamePrefix is greater than that. Am I right? If yes, could you please correct the code to covert that case as well? 
Thanks a lot!

Reply:
You're right. I fixed the code to drop the extension irregardless of the number of dots in the filename. This should do it :)